u2f-gae-demo/war/js/u2f-api.js

```diff
@@ -225,7 +227,7 @@ u2f.WrappedChromeRuntimePort_.prototype.formatSignRequest_ =
  */
 u2f.WrappedChromeRuntimePort_.prototype.formatRegisterRequest_ =
     function(signRequests, registerRequests, timeoutSeconds, reqId) {
-  return {
+    return {
```

Comment:
This file has inconsistent indentation - I'm changing it to 2 space for this segment.